lstchain/scripts/lstchain_mc_dl1ab.py

```diff
@@ -19,31 +32,36 @@
 from lstchain.io import get_dataset_keys, auto_merge_h5files
 from astropy.table import Table
 
-parser = argparse.ArgumentParser(description="Recompute parameters in a DL1 HDF5 file from calibrated images"
-                                             "and based on passed config file. The results are written in a new HDF5 "
-                                             "file."
-                                             "Updated parameters are : Hillas paramaters, wl, r, leakage, "
-                                             "n_islands, intercept, time_gradient")
+parser = argparse.ArgumentParser(
+    description="Recompute DL1b parameters from a DL1a file")
 
 # Required arguments
-parser.add_argument('input_file', type=str, help='path to the DL1 file ')
-
-parser.add_argument('output_file', type=str, help='key for the table of new parameters')
-
-parser.add_argument('--config_file', '-conf', action='store', type=str,
+parser.add_argument('--input-file', '-f', action='store', type=str,
+                    dest='input_file',
+                    help='path to the DL1a file ',
+                    default=None, required=True)
+
+parser.add_argument('--output-file', '-o', action='store', type=str,
+                    dest='output_file',
+                    help='key for the table of new parameters',
+                    default=None, required=True)
+# Optional arguments
+parser.add_argument('--config_file', '--conf', action='store', type=str,
```

Comment:
I think this argument should be `--config-file` to be in agreement with the rest of the changes

Reply:
ctapipe `Tool` uses `--config`, maybe we can bring this in line already

Reply:
well spotted. I'm also moving all `--config-file` to `--config`